.tekton/integration-tests/coo-fbc-upgrade-test-pipeline-4-16.yaml

```diff
@@ -0,0 +1,309 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  annotations:
+    pipelinesascode.tekton.dev/task: "[upgrade, coo-e2e-tests]"
+  name: coo-fbc-upgrade-test-pipeline-416
+spec:
+  description: |
+    This pipeline automates the process of running end-to-end tests for COO
+    using a ROSA (Red Hat OpenShift Service on AWS) cluster. The pipeline provisions
+    the ROSA cluster, installs the COO using with fbc image, runs the tests, collects artifacts,
+    and finally deprovisions the ROSA cluster.
+  params:
+    - name: SNAPSHOT
+      description: 'The JSON string representing the snapshot of the application under test.'
+      default: '{"components": [{"name":"test-app", "containerImage": "quay.io/example/repo:latest"}]}'
+      type: string
+    - name: namespace
+      description: 'Namespace to run tests in'
+      default: 'openshift-cluster-observability-operator'
+      type: string
+    - name: coo_tests_branch
+      description: 'E2E test branch'
+      default: 'release-1.2'
+      type: string
+    - name: coo_version
+      description: 'COO version'
+      default: '1.2.0'
```

Comment:
Where are these parameter values set when the pipeline is called?
Maybe its worth extending https://github.com/rhobs/konflux-coo-fbc/blob/main/hack/customize-pipeline.sh to update these default values? just so we don't forget :)

Reply:
There parameter values are passed in by integration scenario, don't need change here
https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/1d5a566c48b81485d59d1242491b033a2503d445/tenants-config/cluster/stone-prd-rh01/tenants/cluster-observabilit-tenant/1-2/integration-test-fbc-upgrade.yaml#L17